src/main/scala/org/broadinstitute/dsde/rawls/util/WithSupport.scala

```diff
@@ -0,0 +1,54 @@
+package org.broadinstitute.dsde.rawls.util
+
+import org.broadinstitute.dsde.rawls.RawlsExceptionWithErrorReport
+import org.broadinstitute.dsde.rawls.dataaccess.slick._
+import org.broadinstitute.dsde.rawls.dataaccess._
+import org.broadinstitute.dsde.rawls.jobexec.MethodConfigResolver
+import org.broadinstitute.dsde.rawls.jobexec.MethodConfigResolver.MethodInput
+import org.broadinstitute.dsde.rawls.model.{AgoraEntity, UserInfo, ErrorReport, MethodConfiguration}
+import org.broadinstitute.dsde.rawls.webservice.PerRequest.PerRequestMessage
+import spray.http.StatusCodes
+
+import scala.concurrent.ExecutionContext
+import scala.util.{Try, Failure, Success}
+
+//Well, this is a joke.
+trait MethodWiths {
```

Comment:
Were these changed or just moved?  I see the `PerRequest` to `T` changes but anything else?